app/controllers/stories/feeds_controller.rb

```diff
@@ -35,14 +35,20 @@ def assign_feed_stories
     end
 
     def signed_in_base_feed
-      strategy = AbExperiment.get(experiment: :feed_strategy_round_2, controller: self, user: current_user,
-                                  default_value: "original")
-      feed = if strategy.weighted_query_strategy?
-               Articles::Feeds::WeightedQueryStrategy.new(user: current_user, page: @page, tags: params[:tag])
-             elsif Settings::UserExperience.feed_strategy == "basic"
+      feed = if Settings::UserExperience.feed_strategy == "basic"
                Articles::Feeds::Basic.new(user: current_user, page: @page, tag: params[:tag])
              else
-               Articles::Feeds::LargeForemExperimental.new(user: current_user, page: @page, tag: params[:tag])
+               strategy = AbExperiment.get(
+                 experiment: :feed_strategy_round_3,
```

Comment:
With regards to the various areas which need to be flipped from 2 to 3 to 4 going forward, I think we can refactor this so that fewer files are needed to change for a simple test logic update — but I feel like this could be an additional refactor PR out of scope of this one that can be submitted ahead of next round.